drizzlepac/hlautils/astroquery_utils.py

```diff
@@ -44,65 +46,77 @@ def retrieve_observation(obsid, suffix=['FLC'], archive=False,clobber=False):
     """
     local_files = []
 
-    # Query MAST for the data with an observation type of either "science" or "calibration"
-    obsTable = Observations.query_criteria(obs_id=obsid, obstype='all')
+    # Query MAST for the data with an observation type of either "science" or
+    # "calibration"
+    obs_table = Observations.query_criteria(obs_id=obsid, obstype='all')
     # Catch the case where no files are found for download
-    if len(obsTable) == 0:
+    if len(obs_table) == 0:
```

Comment:
```suggestion
    if obs_table:
```
This should preserve the same functionality while being simpler at the same time.